[0m20:45:20.317204 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105b18ec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x12131bd90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x12131bed0>]}


============================== 20:45:20.320508 | ae55db96-85a8-49c4-9916-d2fc0236814a ==============================
[0m20:45:20.320508 [info ] [MainThread]: Running with dbt=1.9.1
[0m20:45:20.320825 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/mclain/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init ae', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:45:20.328207 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/Users/mclain/.dbt')}
[0m20:45:20.328426 [info ] [MainThread]: Creating dbt configuration folder at 
[0m20:45:20.329395 [debug] [MainThread]: Starter project path: /Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/dbt/include/starter_project
[0m20:45:20.332936 [info ] [MainThread]: 
Your new dbt project "ae" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:45:20.333105 [info ] [MainThread]: Setting up your profile.
[0m20:49:12.345939 [info ] [MainThread]: Profile ae written to /Users/mclain/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m20:49:12.391022 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 232.11235, "process_in_blocks": "0", "process_kernel_time": 0.291664, "process_mem_max_rss": "181764096", "process_out_blocks": "0", "process_user_time": 1.024266}
[0m20:49:12.391832 [debug] [MainThread]: Command `dbt init` succeeded at 20:49:12.391666 after 232.11 seconds
[0m20:49:12.392522 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x150ad6d70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x150bb0050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x150b68e20>]}
[0m20:49:12.393100 [debug] [MainThread]: Flushing usage events
[0m20:49:13.025280 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m20:49:26.286167 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106518ec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11081bd90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11081bed0>]}


============================== 20:49:26.289286 | 1b6701ef-e662-4647-8bfa-bc157d2d8fce ==============================
[0m20:49:26.289286 [info ] [MainThread]: Running with dbt=1.9.1
[0m20:49:26.289585 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/mclain/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m20:49:26.297417 [info ] [MainThread]: dbt version: 1.9.1
[0m20:49:26.297579 [info ] [MainThread]: python version: 3.13.0
[0m20:49:26.297699 [info ] [MainThread]: python path: /Library/Frameworks/Python.framework/Versions/3.13/bin/python3
[0m20:49:26.297806 [info ] [MainThread]: os info: macOS-15.1.1-arm64-arm-64bit-Mach-O
[0m20:49:26.728734 [info ] [MainThread]: Using profiles dir at /Users/mclain/.dbt
[0m20:49:26.728962 [info ] [MainThread]: Using profiles.yml file at /Users/mclain/.dbt/profiles.yml
[0m20:49:26.729090 [info ] [MainThread]: Using dbt_project.yml file at /Users/mclain/Documents/Software Engineering/ae-dbt/dbt_project.yml
[0m20:49:26.729392 [info ] [MainThread]: adapter type: snowflake
[0m20:49:26.729553 [info ] [MainThread]: adapter version: 1.9.0
[0m20:49:26.729688 [info ] [MainThread]: Configuration:
[0m20:49:26.729796 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m20:49:26.729897 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m20:49:26.730000 [info ] [MainThread]: Required dependencies:
[0m20:49:26.730139 [debug] [MainThread]: Executing "git --help"
[0m20:49:26.754040 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m20:49:26.754503 [debug] [MainThread]: STDERR: "b''"
[0m20:49:26.754639 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m20:49:26.754773 [info ] [MainThread]: Connection:
[0m20:49:26.754903 [info ] [MainThread]:   account: app
[0m20:49:26.755009 [info ] [MainThread]:   user: CRONIN97
[0m20:49:26.755106 [info ] [MainThread]:   database: AE
[0m20:49:26.755201 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m20:49:26.755295 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m20:49:26.755388 [info ] [MainThread]:   schema: PUBLIC
[0m20:49:26.755481 [info ] [MainThread]:   authenticator: None
[0m20:49:26.755576 [info ] [MainThread]:   oauth_client_id: None
[0m20:49:26.755667 [info ] [MainThread]:   query_tag: None
[0m20:49:26.755756 [info ] [MainThread]:   client_session_keep_alive: False
[0m20:49:26.755849 [info ] [MainThread]:   host: None
[0m20:49:26.755937 [info ] [MainThread]:   port: None
[0m20:49:26.756025 [info ] [MainThread]:   proxy_host: None
[0m20:49:26.756114 [info ] [MainThread]:   proxy_port: None
[0m20:49:26.756201 [info ] [MainThread]:   protocol: None
[0m20:49:26.756289 [info ] [MainThread]:   connect_retries: 1
[0m20:49:26.756377 [info ] [MainThread]:   connect_timeout: None
[0m20:49:26.756463 [info ] [MainThread]:   retry_on_database_errors: False
[0m20:49:26.756549 [info ] [MainThread]:   retry_all: False
[0m20:49:26.756641 [info ] [MainThread]:   insecure_mode: False
[0m20:49:26.756730 [info ] [MainThread]:   reuse_connections: True
[0m20:49:26.757033 [info ] [MainThread]: Registered adapter: snowflake=1.9.0
[0m20:49:26.839670 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m20:49:26.865514 [debug] [MainThread]: Using snowflake connection "debug"
[0m20:49:26.865710 [debug] [MainThread]: On debug: select 1 as id
[0m20:49:26.865837 [debug] [MainThread]: Opening a new connection, currently in state init
[0m20:49:28.652181 [debug] [MainThread]: Snowflake adapter: Error running SQL: select 1 as id
[0m20:49:28.652897 [debug] [MainThread]: Snowflake adapter: Rolling back transaction.
[0m20:49:28.653429 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m20:49:28.653987 [info ] [MainThread]: [31m2 checks failed:[0m
[0m20:49:28.654341 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/mclain/Documents/Software Engineering/ae-dbt/dbt_project.yml> not found

[0m20:49:28.654670 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  250001: Could not connect to Snowflake backend after 2 attempt(s).Aborting

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m20:49:28.658717 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 2.4115496, "process_in_blocks": "0", "process_kernel_time": 0.252094, "process_mem_max_rss": "194002944", "process_out_blocks": "0", "process_user_time": 1.040035}
[0m20:49:28.659249 [debug] [MainThread]: Command `dbt debug` failed at 20:49:28.659115 after 2.41 seconds
[0m20:49:28.659656 [debug] [MainThread]: Connection 'debug' was left open.
[0m20:49:28.659919 [debug] [MainThread]: On debug: No close available on handle
[0m20:49:28.660300 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1177de650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1208439b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x117766cf0>]}
[0m20:49:28.660793 [debug] [MainThread]: Flushing usage events
[0m20:49:29.208099 [debug] [MainThread]: An error was encountered while trying to flush usage events
